pkg/dss/auth/auth.go

```diff
@@ -212,20 +212,18 @@ func (a *Authorizer) AuthInterceptor(ctx context.Context, req interface{}, info
 
 	claims := claims{}
 
-	a.keyGuard.RLock()
```

Comment:
Does the keyGuard relate to the audiences change?  Why did we need it before but not need it now?

Reply:
We didn't need it before since we were only ever setting it in a single thread. @vosst to verify

Reply:
I don't think we need it, but the data race code throws up a red flag since we can access it independently of setting it. so adding it back